### DISCUSSION TITLE
Does Copier work with non-git-controlled directories?

### DISCUSSION THREAD

**Discussion by sharpencrag** [copier-org/copier#1303] [Q&A]
I am running into an error when I try to generate a project from a non-git-version-controlled directory as a template.

Before I put in an issue, I just wanted to confirm that `copier` is _supposed_ to work with other VCS or no VCS at all.

**Comment by sisp** [comment]
Copier works just fine with a template that's not under version control, but updating a project won't work. If you use a VCS for the template, only Git is supported. Updating a project requires both the template and the project to be versioned using Git.

Even when not using Git to version the template, Copier prior to v8.2.0 required a Git client to be installed. Today's release [v8.2.0](https://github.com/copier-org/copier/releases/tag/v8.2.0) removes this requirement if you don't use a Git-managed template (see also #1074).

**Comment by sharpencrag** [comment]
Hm, good to know!

Unfortunately, I don't think this merge actually solves the issue.  Running copier still fails if Git is not installed, after v8.2.0.

Where would be the best place to bring this up?  New issue?  In the PR?

**Reply by yajo** [reply]
I just opened https://github.com/copier-org/copier/issues/1304
